xtrlock.c

```diff
@@ -41,6 +41,8 @@
 #include <shadow.h>
 #endif
 
+//#define DEBUG
```

Comment:
Also have a look at https://stackoverflow.com/questions/1644868/c-define-macro-for-debug-printing/1644898#1644898